### ISSUE TITLE
Dynamically view problem

### ISSUE THREAD

**Issue by schlaincher** [AleSpero/ExpandableCardView#22]
It does not update size  when adding or removing a view to card.

It's a big problem for me.

**Comment by Ayusch** [comment]
Do you mean innerView ?

**Comment by schlaincher** [comment]
Yes. A have a linear layout for innerView. It's my root layout. 
I have to add a view to innerView layout programmatically. I can do this but although the size of the innerVİew is growing, the size of the expandableCardLayout is not growing. So the added views disappear because the card is small.

I have to solve this.
